src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/Requests/PostFileUploadUpdateLogRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests
+{
+    public class PostFileUploadUpdateLogRequest : IPostApiRequest
+    {
+        public string PostUrl => "BulkUpload/UpdateLog";
+
+        public object Data { get; set; }
+
+        public PostFileUploadUpdateLogRequest(FileUploadUpdateLogRequest data)
+        {
+            Data = data;
+        }
+    }
+}
```

Comment:
And this

Reply:
removed as dumped all update endpoint stuff